css-backgrounds-3/Overview.bs

```diff
@@ -1221,6 +1222,7 @@ Value: <<color>>{1,4}
 Initial: (see individual properties)
 Applies to: all elements except [=ruby base containers=] and [=ruby annotation containers=]
 Inherited: no
+Logical property group: border-color
```

Comment:
Only longhands can belong to a logical property group.

Reply:
Thanks. I wonder if I added the field to shorthand tables because of [this recent change](https://github.com/w3c/csswg-drafts/commit/9d5d48d81e352023ff17977969a92ba21810443d#diff-98396261e9414947537a34ccbec62246077c62dc2046cf781b4d3c9fa2cbf958R368-R370).